### PR TITLE
Regenerate CosmosDB SDK

### PR DESCRIPTION
**Pull request by soeom**
Regenerated CosmosDB SDK

### CODE REVIEW

File: sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Generated/Models/CreateUpdateOptions.cs [hunk 1]
```diff
@@ -53,6 +53,7 @@ public CreateUpdateOptions(IDictionary<string, string> additionalProperties = de
         /// Gets or sets unmatched properties from the message are deserialized
         /// this collection
         /// </summary>
+        [JsonExtensionData]
```
Comment (MehaKaushik) [inline sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Generated/Models/CreateUpdateOptions.cs:56]: Run the tests on your local machine, see if they pass, if they fail with a error related to jsonextension data, if they do, remove this.

**Comment by MehaKaushik** [comment]
Add release notes in sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Microsoft.Azure.Management.CosmosDB.csproj

Update the release version like here https://github.com/Azure/azure-sdk-for-net/pull/10928/commits/e8073649f50b330f2d87df3e23a79cde73134b9f

File: sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Generated/Models/CreateUpdateOptions.cs (hunk 1 above)
Comment (soeom) [reply sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Generated/Models/CreateUpdateOptions.cs]: Removed the line. Thanks.

**Comment by soeom** [comment]
@MehaKaushik I made changes following your comment.

**Comment by MehaKaushik** [comment]
Need to change the assembly version in here as well sdk/cosmos-db/Microsoft.Azure.Management.CosmosDB/src/Properties/AssemblyInfo.cs

**Review by MehaKaushik — APPROVED** [review]

**Comment by soeom** [comment]
@isra-fel @erich-wang  Please help merge the PR.

**Review by isra-fel — APPROVED** [review]
LGTM

**Comment by soeom** [comment]
@isra-fel  Please help run CI again

**Comment by isra-fel** [comment]
@dingmeng-xue  could you help force merge this PR? The failed CI "net - core - ci" is for Azure.Core only. It should not block management plane PR. Thanks

**Review by isra-fel — APPROVED** [review]

**Comment by soeom** [comment]
@dingmeng-xue Please help merge the PR

**Comment by dingmeng-xue** [comment]
> @dingmeng-xue could you help force merge this PR? The failed CI "net - core - ci" is for Azure.Core only. It should not block management plane PR. Thanks
I'm not managing this repo and don't have permission. @allenjzhang , do you have permission?

**Comment by soeom** [comment]
@allenjzhang  Can you please help merge the PR?

**Comment by isra-fel** [comment]
Add @kurtzeborn @weshaggard for help force merge.
I've already excluded mgmt plane paths from `net - core - ci` in #11609 but this was created before it.

Thanks

**Pull request merged**